bin/op-angstrom/src/components.rs

```diff
@@ -139,6 +139,28 @@ pub fn initialize_strom_handles<N: NodePrimitives>() -> StromHandles<N> {
     }
 }
 
+// Stub network handle for op-angstrom that doesn't use networking
+#[derive(Clone)]
+struct StubNetworkHandle;
+
+impl NetworkHandle for StubNetworkHandle {
+    type Events<'a> = UnboundedReceiverStream<StromNetworkEvent>;
+
+    fn send_message(&mut self, _peer_id: PeerId, _message: PoolNetworkMessage) {
+        // Op-angstrom doesn't use networking, so this is a no-op
+    }
+
+    fn peer_reputation_change(&mut self, _peer_id: PeerId, _change: ReputationChangeKind) {
+        // Op-angstrom doesn't use networking, so this is a no-op
+    }
+
+    fn subscribe_network_events(&self) -> Self::Events<'_> {
+        // Return an empty receiver since op-angstrom doesn't have network events
+        let (_, rx) = tokio::sync::mpsc::unbounded_channel();
+        UnboundedReceiverStream::new(rx)
+    }
+}
+
```

Comment:
We shouldn't need this, are you doing a follow-up to ensure `RollupPoolManager` doesn't have any network logic with type state?